pkg/json.go

```diff
@@ -102,7 +110,7 @@ func (r *ScorecardResult) AsJSON(showDetails bool, logLevel zapcore.Level, write
 }
 
 // AsJSON2 exports results as JSON for new detail format.
-func (r *ScorecardResult) AsJSON2(showDetails bool, logLevel zapcore.Level, writer io.Writer) error {
+func (r *ScorecardResult) AsJSON2(showDetails bool, logLevel zapcore.Level, checkDocs docs.DocInterface, writer io.Writer) error {
```

Comment:
We can address this in a different PR, but with so many inputs needed to print this object as JSON, I barely think this qualifies to be a class object. We should probably consider trimming down the number of inputs needed or just create a separate util fn for it.

Reply:
sg. I think `docs` could be part of the scorecard structure. The rest is use-define arguments so probably fine to keep as arguments.